packages/nextjs/components/MetaHeader.tsx

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import Head from "next/head";
+
+interface MetaHeaderProps {
+  title?: string;
+  description?: string;
+  favicon?: string;
+  ogTitle?: string;
+  ogDescription?: string;
+  ogImage?: string;
+  twitterCard?: string;
+  twitterTitle?: string;
+  twitterDescription?: string;
+  twitterImage?: string;
+}
+
+const MetaHeader: React.FC<MetaHeaderProps> = ({
+  title,
+  description,
+  favicon,
+  ogTitle,
+  ogDescription,
+  ogImage,
+  twitterCard,
+  twitterTitle,
+  twitterDescription,
+  twitterImage,
+}) => {
+  return (
+    <Head>
```

Comment:
Let's have some defaults (in case the user don't provide it)

title = Scaffold-ETH 2 App
description = Built with 🏗 scaffold-eth-2
image = thumbnail.png